spec/support/ansible_shared/automation_manager/refresher.rb

```diff
@@ -100,12 +100,12 @@
   def assert_counts
     expect(Provider.count).to                                 eq(1)
     expect(automation_manager).to                             have_attributes(:api_version => "3.0.1")
-    expect(automation_manager.configured_systems.count).to    eq(130)
```

Comment:
these counts are a pain.
in aws we changed this to get the counts straight from the API and just assert they are equal to objects counts in the DB.

You would access the API easily via `automation_manager.provider.connect.hosts.count`
Maybe add a safe trap, that the numbers are non-zero